src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -73,6 +108,8 @@
       <PreviouslySourceBuiltNuGetSourceName>previously-source-built</PreviouslySourceBuiltNuGetSourceName>
       <ReferencePackagesNuGetSourceName>reference-packages</ReferencePackagesNuGetSourceName>
       <SourceBuildSources>$(SourceBuildSources);$(PrebuiltNuGetSourceName);$(PreviouslySourceBuiltNuGetSourceName);$(ReferencePackagesNuGetSourceName)</SourceBuildSources>
+      <BuildCommand>$(BuildCommand) /p:RestoreConfigFile=@(NuGetConfigFiles)</BuildCommand>
+      <RestoreConfigFile>@(NuGetConfigFiles)</RestoreConfigFile>
```

Comment:
On the chance that this does work well for multiple config files, I would consider renaming `RestoreConfigFile`->`RestoreConfigFiles`